errors/large-page-data.mdx

```diff
@@ -13,7 +13,7 @@ Reduce the amount of data returned from `getStaticProps`, `getServerSideProps`,
 To inspect the props passed to your page, you can inspect the below element's content in your browser devtools:
 
 ```bash filename="Terminal"
-document.getElementById("__NEXT_DATA__").text
+JSON.parse(document.getElementById("__NEXT_DATA__")?.text)
```

Comment:
```suggestion
JSON.parse(document.getElementById("__NEXT_DATA__").textContent)
```

I think `.text` is non standard. 

Also the optional chaining is not necessary because JSON.parse() will throw if you pass in undefined.